cmd/kluctl/commands/cmd_list_images.go

```diff
@@ -36,7 +37,10 @@ func (cmd *listImagesCmd) Run() error {
 	return withProjectCommandContext(ptArgs, func(ctx *commandCtx) error {
 		result := types.FixedImagesConfig{
 			Images: ctx.images.SeenImages(cmd.Simple),
```

Comment:
I've just realized that the sorting should be part of the above SeenImages function as otherwise the flux-controller would behave differently then the CLI.

Reply:
moved.